sources/packages/web/src/components/common/ProgramDetails.vue

```diff
@@ -79,22 +104,41 @@
       />
     </v-col>
   </v-row>
+  <education-program-deactivation-modal
+    ref="declineEducationProgramModal"
```

Comment:
Ref value seems to be wrong (declineEducationProgramModal)

Reply:
Can you please clarify what is wrong exactly?

Reply:
Got it now, renamed to `deactivateEducationProgramModal`.